### ISSUE TITLE
vr-configengine container build failing

### ISSUE THREAD

**Issue by kalmanoharan** [vrnetlab/vrnetlab#230]
Hello @plajjan ,

Thanks for developing this very useful vrnetlab System. I am really enjoying this and trying to expand further where I can build a topology using topomachine and then push the base start-up configs using vr-config-engine.
While I ran the make command to build the config-engine container, facing the following error.
Can you please help to fix this and guide me to achieve the goal of opology using topomachine and then push the base start-up configs using vr-config-engine.

Specific Error Message:-
```
root@nel17-lab1-vagrantserver1:/home/kalmanoharan/vrnetlab/config-engine-lite# docker run -v $(pwd)/templates:/templates -v $(pwd)/topology:/topology --link SLC-1 --link SLC-2 vr-configengine --topo /topology/lltopo.json  --junos /templates/junos.j2 --run
Unable to find image 'vr-configengine:latest' locally
^C
root@nel17-lab1-vagrantserver1:/home/kalmanoharan/vrnetlab/config-engine-lite# make
docker build -t vr-configengine .
Sending build context to Docker daemon  22.53kB
Step 1/6 : FROM ubuntu:xenial
 ---> 330ae480cb85
Step 2/6 : MAINTAINER Kristian Larsson <kristian@spritelink.net>
 ---> Using cache
 ---> 7a6896675b70
Step 3/6 : ENV DEBIAN_FRONTEND=noninteractive
 ---> Using cache
 ---> 21486a33e1df
Step 4/6 : RUN apt-get update -qy  && apt-get upgrade -qy  && apt-get install -y     bridge-utils     iproute2     libffi-dev     libffi-dev     libjpeg8-dev     libssl-dev     libxml2-dev     libxslt1-dev     libyaml-dev     python-dev     python-dev     python-ipy     python-lxml     python-pip     zlib1g-dev  && rm -rf /var/lib/apt/lists/*  && pip install napalm
 ---> Running in 141d69c52bfe
<< Trimmed >>
Collecting setuptools>=38.4.0 (from napalm)
  Downloading https://files.pythonhosted.org/packages/95/fe/2687907be1d6291a84a9b51220595a4f89d9477032c57f21e7c633465540/setuptools-47.3.1.zip (869kB)
    Complete output from command python setup.py egg_info:
    pkg_resources/py2_warn.py:15: UserWarning: Setuptools no longer works on Python 2
    ************************************************************
    Encountered a version of Setuptools that no longer supports
    this version of Python. Please head to
    https://bit.ly/setuptools-py2-sunset for support.
    ************************************************************
      warnings.warn(pre + "*" * 60 + msg + "*" * 60)

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 32 in /tmp/pip-build-k7CRRj/setuptools/
You are using pip version 8.1.1, however version 20.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
The command '/bin/sh -c apt-get update -qy  && apt-get upgrade -qy  && apt-get install -y     bridge-utils     iproute2     libffi-dev     libffi-dev     libjpeg8-dev     libssl-dev     libxml2-dev     libxslt1-dev     libyaml-dev     python-dev     python-dev     python-ipy     python-lxml     python-pip     zlib1g-dev  && rm -rf /var/lib/apt/lists/*  && pip install napalm' returned a non-zero code: 1
Makefile:11: recipe for target 'all' failed
make: *** [all] Error 1
```

Attached the full logs for your reference.
[config-engine-fail-log.txt](https://github.com/plajjan/vrnetlab/files/4809370/config-engine-fail-log.txt)

**Comment by kalmanoharan** [comment]
@plajjan ,
I think I found the issue in the Docker-file.  Image ubuntu:xenial was using pip 8.1.1 which couldn't install the required NAPALM packages. Therefore I had to upgrade the pip pip install --upgrade pip before running pip install napalm.
Now the able to build the container images successfully.

Will send a pull request to modify the Docker file.

**Comment by mzagozen** [comment]
Fixed by upgrading to Python 3.10.
